doc/releases/changelog-0.28.0.md

```diff
@@ -489,6 +489,9 @@
 * QPE can now accept a target operator instead of a matrix and target wires pair.
   [(#3373)](https://github.com/PennyLaneAI/pennylane/pull/3373)
 
+* The `qml.ops.op_math.Controlled.map_wires` method now uses `base.map_wires` instead of the private `_wires` property setter.
```

Comment:
```suggestion
* The `qml.ops.op_math.Controlled.map_wires` method now uses `base.map_wires` internally instead of the private `_wires` property setter.
```

Reply:
i almost put "internally" here too! I like it